### DISCUSSION TITLE
Record randomly

### DISCUSSION THREAD

**Discussion by lmh6** [gopro/labs#196] [Feature Requests]
Hi guys.. i am trying to find a way to record randomly.   the idea maybe not what gopro wants .. which is to record everything!!

i want it to be a little spontaneous and maybe save some battery as well.  Imagine going on a bike ride and i just want it to randomly start the camera. record for 1 min( or 2). then the camera goes off.  Then after X number of minutes. its come on again and does a 1 min or 2 recording.

The time in between can also be a set amount.  for example.  after 10min, start the camera and record for 2 min. then off the camera... repeat that cycle.

i was looking at this page https://gopro.github.io/labs/control/actions/
Base on the info that was issue there with the following commands
!300SQ - start in approximately 300 seconds.
!60E - end after 60 seconds (End commands are not used for photos.)
!2R - repeat after 2 seconds

!300SQ!60E!300SQ!2R

I wonder if the following command will work... my gopro is not with me at the moment. if any expert would like to chim in on this. it will be greatly appreciated.!

cheers
lmh

**Comment by lmh6** [comment]
i tried the code i set up... but it just turns on and then immediately power off... 

no idea why...anyone?

**Reply by dnewman-gpsw** [reply]
There is a bug in the current release that is messing up these delayed action commands, it will fixed in a few weeks.

**Comment by lmh6** [comment]
Is this only for hero11 or all across rhe board??

**Reply by dnewman-gpsw** [reply]
Just on HERO11.

**Reply by lmh6** [reply]
tried it on a Hero 10.

!300SQ!60E!300SQ!2R

this works as describe.  

now, is there a way to film in random time. :)

**Reply by dnewman-gpsw** [reply]
Not on any shipping Labs firmware, but it will likely in the next HERO11 Labs FW.  No idea is if there is HERO10 release scheduled.

This is how it was prototyped in the QR Command language
!eS  - start at random time within an hour;
!e10E  - end random within 10 seconds.
!e60N - NOP wait for random up to 60seconds

e - is a random number between 0 and 3600 second.
eValue - is a random number between 0 and Value seconds.
